spec/acceptance/rules_spec.rb

```diff
@@ -140,12 +140,20 @@ class { '::firewall': }
         ],
       }
 
+      firewall { '001 ssh allow':
```

Comment:
Sorry that I didn't think about it before, but can you add a comment to this rule explaining that it is specifically to allow for the beaker mgmt connection?

```
# allow all traffic from/to the CI-runner host
```